lms/djangoapps/courseware/module_render.py

```diff
@@ -642,6 +643,8 @@ def rebind_noauth_module_to_user(module, real_user):
 
     system.set(u'user_is_staff', user_is_staff)
     system.set(u'user_is_admin', has_access(user, u'staff', 'global'))
+    system.set(u'user_is_beta_tester', CourseBetaTesterRole(course_id).has_user(user))
```

Comment:
It doesn't look like `has_access` gives you direct information about the beta-tester role (which I think was originally intended just to modify access dates).

My comment on the hangout is that rather than adding this information to the LmsModuleSystem, I'd much rather see the role information added to the user service, and the `days_early_for_beta` attribute added either directly to ORA2 as a field, or to all XBlocks in the platform via an XBlock mixin. We *should not* be adding any new data to ModuleSystem or DescriptorSystem. Those classes are both deprecated.